src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCost = 40;
+  let totalCost = dailyCost * days;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
Instead of using literals for the discount conditions and amounts, define them as constants with descriptive names to make the code more readable and maintainable. For example, use constants like `LONG_TERM_RENTAL_DAYS` and `LONG_TERM_RENTAL_DISCOUNT`.